lua/nvim-treesitter/ts_utils.lua

```diff
@@ -113,6 +113,7 @@ function M.get_named_children(node)
 end
 
 function M.get_node_at_cursor(winnr)
+  if not parsers.has_parser() then return end
```

Comment:
Do we have to add this check to more functions. I guess not (?). This here is only relevant because of the statusline function, right?

Reply:
Yes that's right.